### PR TITLE
fix: Disabling image optimize breaks image path

### LINKED ISSUE #3754: Disabling image optimize breaks image path
When disabling "optimize" the image breaks because the output URL is just the filename with no leading slash or directories.

1. Add image
2. Disable optimize
3. See image break

https://github.com/user-attachments/assets/492623f0-683b-4588-aef2-b56b734742f8

### PR DESCRIPTION
**Pull request by istarkov**
## Description

closes  #3754

## Steps for reproduction

1. click button
2. expect xyz

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env` file

### CODE REVIEW

File: packages/image/src/image-optimize.ts [hunk 1]
```diff
@@ -272,7 +272,11 @@ export const getImageAttributes = (props: {
       src: string;
       srcSet?: string;
       sizes?: string;
-    } = { src: props.src };
+    } = {
+      src: URL.canParse(props.src)
```
[the remark this answers is not in the document]
Comment (kof) [reply packages/image/src/image-optimize.ts:276]: uhh we already started using it everywhere! maybe a polyfill?

[the remark this answers is not in the document]
Comment (TrySound) [reply packages/image/src/image-optimize.ts:276]: It's ok to have in builder but not good in published sites.

[the remark this answers is not in the document]
Comment (istarkov) [reply packages/image/src/image-optimize.ts]: Fixed

File: packages/image/src/image-optimize.ts [hunk 2]
```diff
@@ -232,6 +232,18 @@ const DEFAULT_SIZES = "(min-width: 1280px) 50vw, 100vw";
 
 const DEFAULT_QUALITY = 80;
 
+/**
+ * URL.canParse(props.src)
+ */
+export const UrlCanParse = (url: string) => {
```
Comment (kof) [inline packages/image/src/image-optimize.ts:238]: can we please have a better function name? "canParseUrl" would be nice

**Review by kof — APPROVED** [review]

File: packages/image/src/image-optimize.ts (hunk 2 above)
Comment (TrySound) [reply packages/image/src/image-optimize.ts:238]: Current one is catchy 😉

**Pull request merged**